lib/src/kvs/tests/cluster_init.rs

```diff
@@ -166,6 +168,7 @@ async fn single_live_queries_are_garbage_collected() {
 		cond: None,
 		fetch: None,
 		archived: None,
+		session: Some(Value::None),
```

Comment:
Shouldn't this be `None` instead of `Some`? I understand it's indicating the session is set. But shouldn't Session be a Session type? Instead of a Value type. I imagine it will result in weird implementations in the future, or bugs such as people assuming Some means ses!=Value::None.

Reply:
It broke some of your tests. The `session` is empty, but the session does actually need to be there for testing. When running in real life (not in tests), the session is always set on the LIVE statement, so I just set it to an empty object. We definitely want to improve tests in the future for this PR 👍 .

Reply:
I think if having it as None breaks the tests then either the tests need to be fixed or we need to assume it isn't always set. We should really have tests that reflect the real state of the db and we should have implementations that don't work around failing tests. Let's try to fix this after 1.0